src/interfaces/user.ts

```diff
@@ -1,10 +1,13 @@
 import { IAddress } from './address';
 
-export interface IProfile {
-  firstname: string;
-  lastname: string;
-  gender: string;
-  dateOfBirth: number;
+export interface IUserBio {
+    firstname: string;
+    lastname: string;
+    gender: string;
+    dateOfBirth: number;
+}
+
+export interface IProfile extends IUserBio {
```

Comment:
Since the bio info has been extracted from the `IProfile`, It might be better to call this `ITenantProfile.`

Also, I don't think `IProfile` should extend `IUserBio` - it should hold just the information required for a tenant profile.